README.md

```diff
@@ -10,7 +10,7 @@
 - [Google group](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
 - [Slack channel](https://kubernetes.slack.com/messages/C2C40FMNF/)
 - [Events and meetings calendar](https://calendar.google.com/calendar/embed?src=coreos.com_regcvcrgvq98lua2ikijg1g1uk%40group.calendar.google.com&ctz=America/Los_Angeles)
-- [Meeting agenda and notes](https://docs.google.com/document/d/1Q6Ux-vsiRuiXhXSkVqGr9FUZifqVGRdCWkzVI1UT4l0/edit?usp=sharing) ([read-only archive](https://docs.google.com/a/google.com/document/d/e/2PACX-1vRsGZTlmggTI5T4ysYGlaJiaW3vgnrc_dlv-B1nnt69gU2Xt6vzRpZDMm-7driIj14T1DraicGzkAHr/pub))
+- [Meeting agenda and notes](https://docs.google.com/document/d/1Fu6HxXQu8wl6TwloGUEOXVzZ1rwZ72IAhglnaAMCPqA/edit) ([read-only archive](https://docs.google.com/a/google.com/document/d/e/2PACX-1vRsGZTlmggTI5T4ysYGlaJiaW3vgnrc_dlv-B1nnt69gU2Xt6vzRpZDMm-7driIj14T1DraicGzkAHr/pub))
```

Comment:
I am not able to access the read-only archive even though I am a part of https://groups.google.com/forum/#!forum/kubernetes-sig-release. What am I doing wrong?

Reply:
I don't have access to that either?!

Reply:
I tried updating permissions for kubernetes-sig-release@googlegroups.com. It is a "Google" doc, and can't be publicly shared. Please let me know if that works. If not, we'll try something else.

Reply:
> Please let me know if that works. If not, we'll try something else.

Doesn't work. :(